markdown/blog/2024-marketing-report.md

```diff
@@ -0,0 +1,196 @@
+
+## Accomplishments and Milestones
+We’ve focused on laying the groundwork for long-term growth and engagement for the last six months. Our marketing initiatives have been designed to elevate AsyncAPI’s visibility, simplify our communication strategy, and ensure we’re prepared to grow alongside our community. 
+
+Here’s a summary of what we’ve done so far:
+- Launched targeted social media campaigns that reached 5,031 impressions, supported by our community of designers:
+  - #AsyncAPIParis2024 campaign 
+  - #YourAsyncAPIStory celebrating AsyncAPI at 8 
+
+- Held successful [online event](https://www.youtube.com/watch?v=XGn8v0yBfOI) that engaged and informed over 350 community members and prospective contributors
+
+- Implemented regular content updates on our platforms to keep followers informed and engaged
+
+- Proposed the Student Ambassador Program initiative that aims to bring more attention to AsyncAPI's mission and tools, further building our presence and strengthening community ties
+
+## Implementing Buffer for Social Media Management
+
+To improve the efficiency and security of our social media management, we’ve introduced a paid version of Buffer for posting content across our platforms. This change enables us to streamline content scheduling, ensuring consistent, professional updates without compromising security. Importantly, using Buffer restricts access to the main social media accounts, adding a layer of protection and allowing us to manage permissions efficiently.
+
+Here are some of the benefits of using the Buffer subscription
+- **Centralized Dashboard:** Manage all social media platforms from a single interface, making tracking and organizing content easier. 
+- **Improved Security:** No need for multiple people to log into the actual social media accounts. This minimizes the risks of unauthorized changes or breaches.
+- **Better Analytics and Insights:** Access in-depth analytics to understand which posts perform and engagement metrics best and adjust strategies accordingly. 
+- **Content Approval Workflow:** Allow team members to draft posts for review, ensuring high-quality content that aligns with AsyncAPI’s voice.
+
+
+Please note that I currently do not have direct access to AsyncAPI’s social media accounts. Buffer provides an effective solution for posting content, monitoring performance, and maintaining the high standards of AsyncAPI's public presence without broad access.
+
+As part of the benefit of using Buffer, we have access to our social media analytics. Below are the analytics for LinkedIn and X (formally Twitter) from January 1st to December 12th, 2024.
+
+
+
+<Figure
+  src="/img/posts/marketing-images/linkedIn-metrics-insights.webp"
+  alt="Graph showing AsyncAPI LinkedIn metrics including follower growth, engagement rates, and post performance over time"
+  caption="AsyncAPI LinkedIn graph showing metrics insights from Buffer analytics."
```

Comment:
the graph is related to followers growth, the other image is about more. But the diagram itself is just followers, so please make it clear in `alt` and `caption`